src/lib/components/CreatibutorsField.js

```diff
@@ -8,7 +8,12 @@ import _get from 'lodash/get';
 import React, { Component } from 'react';
```

Comment:
Is this file still valid? There is a file of same name in `src/lib/components/Creatibutors/CreatibutorsField.js` above. I am assuming this one can be deleted.

Reply:
Removed!